### ISSUE TITLE
Add docs in the new docs folder

### ISSUE THREAD

**Issue by raix** [CollectionFS/Meteor-CollectionFS#555]
If you have ideas for documentation and want to contribute to the project lets know

**Comment by Ethaan** [comment]
Hi @raix i just made this pretty basic FSCollection example using (gridFS), its a pretty basic demo online, i think it could be a good point to newcomers.

**Comment by raix** [comment]
Wow @Ethaan Looks really nice!
Specially when you get the first image (have to be png) uploaded you get to see the code examples on filters etc.

@aldeed check it out

**Comment by aldeed** [comment]
Yeah, lots of good examples. We should have a list of demo links in the docs.

**Comment by Ethaan** [comment]
Thanks @raix  Yep i try to put a mini-modal on each section(download file, filters,deletes) just to point newcomers how it works and also show how it works.
